### ISSUE TITLE
Subtracting variable doubles the variable value

### ISSUE THREAD

**Issue by maggiedeuitch** [rafaelvascc/google-meet-dice-roller#4]
Hello, I'm having trouble with subtracting variables in my dice set commands.

I have a variable `"variables.check_penalty": "4"`
I've set my command (test_subtract) as:

`1d20 -{check_penalty}`

This results in the `check_penalty` value being doubled:

```
Morgan.test_subtract
1d20:
	 (3) Total: 3
Total: 3 - 8 = -5
```
I would expect it to be:
```
Morgan.test_subtract
1d20:
	 (3) Total: 3
Total: 3 - 4 = -1
```
However, if I don't use a variable in my command, then it works as expected:
`1d20 -4` 
```
Morgan.test_subtract
1d20:
	 (20) Total: 20
Total: 20 - 4 = 16
```

I also tried setting the variable to `-4` and then the command as `1d20+{check_penalty}` and this does not return anything. With a space after the dice part  `1d20 +{check_penalty}` it ignores the variable:
```
Morgan.test_subtract_2
d20:
	 (9) Total: 9
```

**Comment by rafaelvascc** [comment]
For now use `1d20-{check_penalty}` (without spaces).
I will take a look into this bug tomorrow.

**Comment by maggiedeuitch** [comment]
Unfortunately, `1d20-{check_penalty}`  does not return anything:

```
You 3:49 PM
Morgan.test
```

**Comment by rafaelvascc** [comment]
That's odd...
Anyway, i will take a look into it

**Comment by rafaelvascc** [comment]
Fixed in version 1.1.1. Version will be live in some days, we need to wait Google to approve it.

**Comment by maggiedeuitch** [comment]
@rafaelvascc Looks good, I can now use the variable with subtractions. Thank you!
